src/dso_api/dynamic_api/routers.py

```diff
@@ -48,14 +48,21 @@
     from schematools.contrib.django.models import DynamicModel
 
 
-class DynamicAPIRootView(APIIndexView):
+class DynamicAPIIndexView(APIIndexView):
```

Comment:
There's no need to rename this class, it is still Root view.

Reply:
But its now also an index view at locations other than root

Reply:
Those are roots of subtrees, right? :)

Reply:
So it should be called DynamicAPINodeView? or DynamicAPISubTreeView :-)

Reply:
Why not keep DynamicAPIRootView? It's just fine.